tools/java/pom.xml

```diff
@@ -0,0 +1,219 @@
+  </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>29.0-jre</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.ant</groupId>
+      <artifactId>ant</artifactId>
+      <version>1.10.6</version>
+      <scope>compile</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>xerces</groupId>
+      <artifactId>xercesImpl</artifactId>
+      <version>2.9.0</version> <!-- there are later ones! --> 
+    </dependency>
+    
+    <dependency>
+      <groupId>com.google.myanmartools</groupId>
+      <artifactId>myanmar-tools</artifactId>
+      <version>1.1.1</version> <!-- there's a 1.1.2 ! -->
+    </dependency>
+  </dependencies>
+
+  <build>
+    <sourceDirectory>.</sourceDirectory> <!-- TODO: HACK! until we can pivot src dirs -->
+   <resources>
+     <resource>
+       <directory>.</directory>
+        <includes>
+          <include>org/unicode/cldr/util/data/**/*</include>
+          <include>org/unicode/cldr/icu/*.txt</include>
+          <include>org/unicode/cldr/icu/*.html</include>
+          <include>org/unicode/cldr/json/*.txt</include>
+          <include>org/unicode/cldr/tool/*.txt</include>
+          <include>org/unicode/cldr/tool/*.css</include>
+          <include>org/unicode/cldr/tool/*.html</include>
+          <include>org/unicode/cldr/tool/*.xml</include>
+          <!-- <include>org/unicode/**/package.html</include> --> <!-- TODO: not working.-->
+        </includes>
+        <!-- <excludes>
+          <exclude>target</exclude>
+          <exclude>com</exclude>
+          <exclude>classes</exclude>
+          <exclude>libs</exclude>
+        </excludes> -->
+     </resource>
+   </resources>
+
+    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
+      <plugins>
+        <!-- clean lifecycle, see https://maven.apache.org/ref/current/maven-core/lifecycles.html#clean_Lifecycle -->
+        <plugin>
+          <artifactId>maven-clean-plugin</artifactId>
+          <version>3.1.0</version>
+        </plugin>
+        <!-- default lifecycle, jar packaging: see https://maven.apache.org/ref/current/maven-core/default-bindings.html#Plugin_bindings_for_jar_packaging -->
+        <plugin>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>3.0.2</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.8.0</version>
+          <configuration>
+            <!-- TODO: pivot this when we can restructure src dirs -->
```

Comment:
It sounds like there's a vaguely structured plan behind some of these TODOs. Having a CLDR ticket for the discussion would be great.

Reply:
I will update CLDR-11585